feature-libs/organization/administration/components/unit/details/unit-details.component.ts

```diff
@@ -18,13 +16,4 @@ import { UnitItemService } from '../services/unit-item.service';
   ],
   host: { class: 'content-wrapper' },
 })
-export class UnitDetailsComponent {
-  model$: Observable<B2BUnit> = this.itemService.key$.pipe(
-    switchMap((code) => this.itemService.load(code)),
-    startWith({})
-  );
-  isInEditMode$ = this.itemService.isInEditMode$;
-
-  constructor(protected itemService: ItemService<B2BUnit>) {}
-  iconTypes = ICON_TYPE;
```

Comment:
`iconTypes = ICON_TYPE;` looks like mistake added in some previous PR (not released yet)